Classes/DataProcessing/ListItemsDataProcessor.php

```diff
@@ -50,7 +50,7 @@ public function process(
         $processedRecordVariables = [];
         foreach ($items as $key => $record) {
             $recordContentObjectRenderer = GeneralUtility::makeInstance(ContentObjectRenderer::class);
-            $recordContentObjectRenderer->start($record, 'tx_listelements_item', $request);
+            $recordContentObjectRenderer->start($record, 'tx_listelements_item');
```

Comment:
we then need $recordContentObjectRenderer->setRequest() though then.

Reply:
👍 , done